src/engine/ChromaticFilter.cs

```diff
@@ -0,0 +1,34 @@
+using Godot;
+
+/// <summary>
+/// A chromatic aberration and barrel distrortion filter effect
```

Comment:
`///   A chromatic aberration and barrel distrortion filter effect`
2 extra spaces